web/packages/teleport/package.json

```diff
@@ -32,11 +32,12 @@
     "@opentelemetry/sdk-trace-base": "1.26.0",
     "@opentelemetry/sdk-trace-web": "1.26.0",
     "@opentelemetry/semantic-conventions": "1.27.0",
-    "@xterm/xterm": "^5.5.0",
     "@xterm/addon-canvas": "^0.7.0",
     "@xterm/addon-fit": "^0.10.0",
+    "@xterm/addon-image": "^0.8.0",
```

Comment:
Did you have a chance to look into how this works?

In theory, it's something that would be nice to have in Connect too. But it also would be the first time we render 3rd-party content within Connect. If it leads to an RCE, our security posture is definitely much worse than that of a regular web browser.